### DISCUSSION TITLE
Unable to pip install cuquantum on Windows due to conflicting dependencies error

### DISCUSSION THREAD

**Discussion by foongminwong** [NVIDIA/cuQuantum#128] [Q&A]
We tried `pip install cuquantum`, then the installation failed and threw the following error: `ERROR: Cannot install cuquantum==23.10.0, cuquantum==23.3.0, cuquantum==23.6.0, cuquantum==23.6.1 and cuquantum==24.3.0 
because these package versions have conflicting dependencies.`

Below shows the full error message:
![image](https://github.com/NVIDIA/cuQuantum/assets/23441795/7717f1bb-622b-4f4e-8ef3-893e67899e49)

Does anybody know how to resolve this?

Python Version: 3.10.5
OS: Windows 10 (with NVIDIA GPU)

**Comment by A-tA-v** [comment]
Same issue on OSX Sonoma, Python 3.12, no CUDA GPU (macbook pro)

**Comment by A-tA-v** [comment]
It seems that installation is unable to find CUDA libraries. 
```
  ************************************************************************
  WARNING: Unable to detect NVIDIA CUDA Toolkit installation, assuming
  WARNING: CUDA 11 is in use... This behavior is subject to future change.
  WARNING: To be explicit, instead of "pip install cuquantum-python" run
  WARNING: "pip install cuquantum-python-cuXX", with XX being the major
  WARNING: version of your CUDA Toolkit installation.
  ************************************************************************
```
The question: is it possible to install CUDA toolkit without CUDA hardware? 
The old answers suggested some workarounds: https://stackoverflow.com/questions/3087361/gpu-emulator-for-cuda-programming-without-the-hardware
It turns out there is no a good solution ... so no cuQuantum at the moment.

**Comment by yangcal** [comment]
As of cuQuantum 24.06, `Windows` is not an officially supported OS. Meanwhile, an NVIDIA GPU with compute capability 7.0+ is required. For more details, please refer to [cuTensorNet](https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html) and [cuStateVec](https://docs.nvidia.com/cuda/cuquantum/latest/custatevec/index.html) doc page

**Reply by foongminwong** [reply]
Thank you, how about Ubuntu (WSL2)?

**Reply by yangcal** [reply]
Since Ubuntu WSL2 has full linux kernels, my guess is that it should work though we haven't tested it. Feel free to share the outcome with us.
